tests/conftest.py

```diff
@@ -1,7 +1,7 @@
 from pathlib import Path
 
 import pytest
-import yaml
+from ruamel.yaml import YAML
```

Comment:
We've been depending on `pyyaml` for tests by accident, and did not notice since it got installed together with pre-commit. Let's use ruamel since that's what is used in the aiidalab package.